gax-java/gax/src/test/java/com/google/api/gax/rpc/EndpointContextTest.java

```diff
@@ -339,6 +340,7 @@ void endpointContextBuild_gdchFlow_noUniverseDomain_customEndpoint() throws IOEx
   // For this test running locally or in CI, check that the Env Var is set properly.
   // This test should only run when the maven profile `EnvVarTest` is enabled.
```

Comment:
Can you remove this line: `// This test should only run when the maven profile `EnvVarTest` is enabled.`

Reply:
Done.